dashboard/test/models/pd/workshop_material_order_test.rb

```diff
@@ -301,4 +301,18 @@ class Pd::WorkshopMaterialOrderTest < ActiveSupport::TestCase
     order.refresh
     assert_nil order.tracking_id
   end
+
+  test 'orders are still valid if the user is deleted' do
```

Comment:
I'll remove the initial assert. As far as 2 tests, would you prefer to separate them? That seems extra cumbersome, given that we don't (yet?) support describe/it syntax with nested setups.